whisker-backend/pkg/apis/v1/flows.go

```diff
@@ -0,0 +1,48 @@
+package v1
+
+import "time"
+
+const (
+	FlowsPath       = sep + "flows"
+	FlowsStreamPath = FlowsPath + sep + "_stream"
```

Comment:
what does the `_` signify our APIs?

Reply:
Generally the API for list and get would be `/resources` and `/resources/{id}`, however, `_stream` wouldn't signal a resource ID. For these cases I like to prefix the path with an underscore so that it would likely never conflict with any sort of resource identifier.

Reply:
Kubernetes API uses a pattern like this:

```
GET /resources?watch=true
```

Seems like params might make more sense here instead of using a special value `{id}`?